schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    id INT,
```

Comment:
- [ ] [OPTIONAL] Kindly add ``PRIMARY KEY (id)` as well.

Reply:
Thanks for the wise feedback 😄  ...I will add the primary key 👍